src/Microsoft.Diagnostics.Monitoring.Options/StorageOptions.cs

```diff
@@ -18,10 +18,17 @@ public class StorageOptions
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_StorageOptions_DumpTempFolder))]
         public string? DumpTempFolder { get; set; }
 
-        [Experimental]
+        [Options.Experimental]
         [Display(
             ResourceType = typeof(OptionsDisplayStrings),
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_StorageOptions_SharedLibraryPath))]
         public string? SharedLibraryPath { get; set; }
+
+        internal bool Configured
+        {
+            [MemberNotNullWhen(true, nameof(DumpTempFolder))]
```

Comment:
note: This support specifying multiple members, e.g.
`[MemberNotNullWhen(true, nameof(DumpTempFolder), nameof(SharedLibraryPath)]`